client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/configure-sms-notification/phone-number-editor.tsx

```diff
@@ -26,10 +135,62 @@ export default function PhoneNumberEditor( { toggleModal }: Props ) {
 		>
 			<div className="notification-settings__sub-title">{ subTitle }</div>
 
-			<form onSubmit={ onSave }>
-				{
-					// Add form fields here
-				 }
+			<form className="configure-contact__form" onSubmit={ onSave }>
```

Comment:
I can see that both Email and Phone number editors share a similar structure (modal, title, subtitle, etc.) I think there is an opportunity for us to create a reusable Modal that abstracts the common structure here.

I am thinking like the one below. wdyt? 

```
<FormModal title={ title } onSubmit={ onSave } disableSubmit={ ! smsItem.name || ! smsItem.countryCode || ! smsItem.phoneNumber }>
	<FormFieldset>
		<FormLabel htmlFor="name">{ translate( 'Name' ) }</FormLabel>
		...
	</FormFieldset>
	...
</FormModal>
```

Reply:
I was thinking the same, mainly because this component has gotten very large. I think it's better to wrap this in a different PR. WDYT?

Reply:
Yes, I agree. We can do this in another PR 👍